SmartContract/migrations/2_deploy_contracts.js

```diff
@@ -1,6 +1,7 @@
 const Verifier = artifacts.require("Verifier");
 const UnisonToken = artifacts.require("UnisonToken");
 const RandomSource = artifacts.require("Utilities/RandomSource.sol");
+const Faucet = artifacts.require("Faucet.sol")
```

Comment:
@Kaleb-Bruwer are these changes fine?

Reply:
These are my changes, which are already on dev. I don't know why git marked it as a change